client/verta/verta/_tracking/experimentrun.py

```diff
@@ -272,9 +302,16 @@ def _upload_artifact(self, key, artifact_stream, part_size=64*(10**6)):
         url_for_artifact = self._get_url_for_artifact(key, "PUT", part_num=1)
 
         if url_for_artifact.multipart_upload_ok:
+            file_parts = iter(
+                lambda: artifact_stream.read(_artifact_utils.MULTIPART_UPLOAD_PART_SIZE), b'')
```

Comment:
Very minor: End `)` on new line is preferred
```suggestion
            file_parts = iter(
                lambda: artifact_stream.read(_artifact_utils.MULTIPART_UPLOAD_PART_SIZE), b'',
            )
```

Reply:
the line became shorter